packages/components/src/components/scroller/scroller.tsx

```diff
@@ -108,19 +109,20 @@ export class Scroller {
       const PrefixedTagNames = getPrefixedTagNames(this.host);
       return (
         <div key={direction} class={direction === 'next' ? 'action-next' : 'action-prev'}>
-          <PrefixedTagNames.pButtonPure
+          <button
             class="button"
             type="button"
             tabIndex={-1}
-            hideLabel={true}
-            size="inherit"
-            icon={direction === 'next' ? 'arrow-head-right' : 'arrow-head-left'}
             onClick={() => this.scrollOnPrevNextClick(direction)}
-            theme={this.theme}
             aria-hidden="true"
+            aria-label={direction}
           >
-            {direction}
-          </PrefixedTagNames.pButtonPure>
+            <PrefixedTagNames.pIcon
+              class="icon"
+              name={direction === 'next' ? 'arrow-head-right' : 'arrow-head-left'}
```

Comment:
Why do we stick to arrow heads here when everywhere else we have arrows?

Reply:
We checked this in the beginning, its to prominent. Design decision.